Source/Widgets/Viewer/Viewer.js

```diff
@@ -401,11 +405,27 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
 
         var scene3DOnly = defaultValue(options.scene3DOnly, false);
 
+        if (defined(options.clock)) {
+            deprecationWarning('Viewer.options.clock', 'Passing options.clock when creating a new Viewer instance was deprecated in Cesium 1.34 and will be removed in Cesium 1.37, pass options.clockViewModel instead.');
+        }
+
+        var clock;
+        var clockViewModel;
+        var destroyClockViewModel = false;
+        if (defined(options.clockViewModel)) {
+            clockViewModel = options.clockViewModel;
+            clock = clockViewModel.clock;
+        } else {
+            clock = new Clock();
```

Comment:
Shouldn't this be `defaultValue(options.clock, new Clock())` to maintain the existing behavior until `options.clock` is removed?